lib/generators/v3/index.js

```diff
@@ -0,0 +1,239 @@
+'use strict';
+const Chance = require('chance').Chance();
+const Format = require('./format');
+const Randexp = require('randexp').randexp;
+
+const mock = (node, useExample)  => {
+    let mock;
+
+    // Parameters and responses live inside the schema keyword now
+    const schema = node.schema;
+
+    if (schema) {
+        let type = schema.type || findType(schema);
+        let example = schema.examples || schema.example || schema.default;
```

Comment:
Added default as an example, which the spec says can be used if no examples exist.

This `schema.example || ...` might give funny results if example is 0 or false.